comp/dogstatsd/listeners/uds_common.go

```diff
@@ -144,11 +144,25 @@ func NewUDSListener(packetOut chan packets.Packets, sharedPacketPoolManager *pac
 
 // Listen runs the intake loop. Should be called in its own goroutine
 func (l *UDSListener) handleConnection(conn *net.UnixConn) error {
+	listenerID := l.getListenerID(conn)
+	tlmListenerID := listenerID
+	if !l.config.GetBool("dogstatsd_telemetry_enabled_listener_id") {
+		tlmListenerID = ""
```

Comment:
I think I would've done it differently: I would be testing the telemetry flag (either passed as ar parameter or calling `l.config` directly, no preference) in `getListenerID()` directly, this way, I would return either:
  * if false: 
```go
"uds-" + conn.LocalAddr().Network()
```
  * if true: 
```go
"uds-" + conn.LocalAddr().Network() + "-" + connID
```

Even if not separating the information per connection ID, it will at least attach enough information to differentiate between `unix` and `unixgram`, WDYT? Let me know if you think it has no value, since current implementation looks good as well.